### PR TITLE
Vet_Clinic Database Group-Task

### PR DESCRIPTION
**Pull request by Josphat205**
## Procedures

- Added an extra column to the owners table
-  Added data to the database
- Run queries to check execution time
## Queries and Screenshots

- Run queries to check execution time
### **Screenshot 3**

- Added commands to improve execution time

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -71,4 +71,10 @@ INNER JOIN vets v ON v.id = vet_id WHERE v.id = 2;
...
 INNER JOIN animals ON species.id = animals.species_id
 INNER JOIN visits ON animals.id = animal_id
-INNER JOIN vets v ON v.id = vet_id WHERE v.id = 2 GROUP BY species.name ORDER BY count DESC;
\ No newline at end of file
+INNER JOIN vets v ON v.id = vet_id WHERE v.id = 2 GROUP BY species.name ORDER BY count DESC;
+
+
+
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animal_id = 4;
+EXPLAIN ANALYZE SELECT * FROM visits where vet_id = 2;
```
Comment (mugishasam123) [inline queries.sql:79]: - [x] Kindly you should do as much as possible to normalize your database performance by reducing the execution time after improving performance.  the project request that you should get execution time after normalization which is under 1 sec (1000ms).

your execution time(after improving performance):
image

**Review by mugishasam123 — CHANGES_REQUESTED** [review]
Hi @Josphat205  👋 , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>
alt

## Highlights
- No linter errors 👍 
- Nice use of git-flow 🥇 
- Descriptive commit messages, keep rocking 💯 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Josphat205) [reply queries.sql:79]: ## Improvement of performance
Capture

**Review by omarsalem7 — APPROVED** [review]
Hi Team 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**